packages/cli/src/index.ts

```diff
@@ -78,8 +80,16 @@ cli.command('serve', 'serve the app locally')
     .option('--allowExternal, --allow-external', '[boolean] allow external IPs to access the server', {
         default: false,
     })
+    .option('--appType [appType], --app-type', '[string] specify app type. Overrides manifest values')
```

Comment:
Is there a reason why `[appType]` is in brackets? 🤔 

Should it perhaps be:
`.option('--appType, --app-type, <appType>', ...` instead? (taking inspiration from the `entryPath`)

Reply:
@julianiff Still open?

Reply:
ahh brackets means its optional. sorry we cleared it in a 1on1. 
From [cac npm](https://www.npmjs.com/package/cac#brackets): 
When using brackets in command name, angled brackets indicate required command arguments, while square bracket indicate optional arguments.